### PR TITLE
OHIE-327 - Additional dev and user docs

### PR DESCRIPTION
**Pull request by rcrichton**
Adds docs on how to set the go app setup for development and user doc on how to run instant OpenHIE.

This is for OHIE-327 (the ticket didn't exist when I first created the branch)

### CODE REVIEW

File: docs/docs/introduction/running.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+---
+id: running
+title: Running Instant OpenHIE
```
Comment (BMartinos) [inline docs/docs/introduction/running.md]: Should this not rather read `Running InstantHIE`?

Comment (BMartinos) [inline docs/docs/introduction/running.md]: Should this not rather be `Running InstantHIE`?

Comment (rcrichton) [reply docs/docs/introduction/running.md]: I don't think the name is really decided yet. I see we are using a mix of the two in the docs at the moment.

File: docs/docs/introduction/running.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+sidebar_label: Running
+keywords:
+  - InstantHIE
+  - Running Instant OpenHIE
+description: How to run Instant OpenHIE
+---
+
+Instant OpenHIE is designed to be easily downloaded and run my anyone on any platform. To get started you will need to first install docker and docker-compose which Instant OpenHIE uses to run all of the necessary services and applications. The following links guide you on how to do this for your platform:
```
Comment (BMartinos) [inline docs/docs/introduction/running.md]: Typo: `... and run by anyone...`

File: docs/docs/introduction/running.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+* [Docker engine](https://docs.docker.com/install/)
+* [Docker compose](https://docs.docker.com/compose/install/)
+
+Once these are installed you can download the latest Instant OpenHIE executable from github. Look for the latest release and download the executable that matches your platform under the assets section of the release:
+
+* [Instant OpenHIE](https://github.com/openhie/instant/releases)
+
+Once the download is complete, move it somewhere memorable. From there you may execute it and you will be presented with a commandline interface. Use the arrow keys to navigate it and choose 'Start Instant OpenHIE'. This will initiate some download that my take some time. Once completed it will then startup the Instant OpenHIE services/application. A web page will be opened with links to the started services/application as well as links to their documentation.
+
+Congratulations, you have successfully started up Instant OpenHIM. Follow the instruction on the web page to stop the services when you are done.
```
Comment (BMartinos) [inline docs/docs/introduction/running.md]: Typo: `Instant OpenHIM` :)

File: goinstant/README.md [hunk 4]
```diff
@@ -1,12 +1,23 @@
...
-This is a Go app and can be built as a native binary for any operating system. Static assets and templates are built using packr2. 
+This is a Go app and can be built as a native binary for any operating system. Static assets and templates are built using packr2.
 
-Run it using `go run goinstant.go` or build the binary using `go build`. To build releases, create a tag and upload the binaries built. A convenience bash script is included to build binaries. 
+## Dev prerequisites
+
+* Install go, [see here](https://golang.org/doc/install). For Ubuntu you might want to use the go snap package, [see here](https://snapcraft.io/install/go/ubuntu).
+* Install packr2: **Outside** of the goinstant folder (so that it doesn't get installed as a module) run: `go get -u github.com/gobuffalo/packr/v2/packr2`
+* Add go binaries to you system $PATH, on ubuntu: Add `export PATH=$PATH:$HOME/go/bin` to the end of your ~/.bashrc file. To use this change immediately source it: `source ~/.bashrc`
+* install dependencies, run this from the goinstant folder: `go get`
```
Comment (BMartinos) [inline goinstant/README.md]: Capitalise this first character of the first word

File: goinstant/README.md [hunk 5]
```diff
@@ -1,12 +1,23 @@
...
+
+## Running and building
+
+Run the app using `go run goinstant.go` or build the binary using `go build`. To build releases, create a tag and upload the binaries built. A convenience bash script is included to build binaries.
+
+Note: this script won't work if you have a `goistant/data/` folder that gets created when upping the docker-compose files through the go app. Delete this first: `sudo rm -r goisntant/data`
```
Comment (BMartinos) [inline goinstant/README.md]: `Upping` -> `when starting up`
Typo: `goinstant` is spelt `goisntant`

File: docs/docs/introduction/running.md (hunk 1 above)
Comment (BMartinos) [reply docs/docs/introduction/running.md]: Ok cool makes sense, we can create a ticket to update this through when a proper name is decided :+1:

Comment (BMartinos) [reply docs/docs/introduction/running.md]: This has already been answer in a previous duplicate question

**Pull request merged**